CPP_STYLE_GUIDE.md

```diff
@@ -195,6 +196,12 @@ class FancyContainer {
 
 Further reading in the [C++ Core Guidelines][ES.47].
 
+### Use explicit pointer comparisons
+
+Use explicit comparisons to `nullptr` when testing pointers, i.e.
+`if (foo == nullptr)` instead of `if (foo)` and
+`foo != nullptr` instead of `!foo`.
```

Comment:
For consistency, maybe make this `if (foo != nullptr)` and `if (!foo)`.

Reply:
@cjihrig yup, done!